src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,21 @@
+import { dispatch } from "utils/dispatch";
+
+export class LoadingIndicator {
+  constructor() {
+    this.bindListeners();
+  }
+
+  bindListeners() {
+    dispatch.on("show-loading-indicator", this.show);
+    dispatch.on("hide-loading-indicator", this.hide);
+  }
+
+  show() {
+    // the default for the corresponding DOM element is false
+    document.getElementById("loading-container").setAttribute("hidden", false);
```

Comment:
@timhitchins you could add the HTML node as a class property to be a little more DRY here, e.g.

```js
export class LoadingIndidcator {
  node = document.getElementById("loading-container");
// ...
```

Reply:
dry is def better. good one.